From fa6d8bfa26ce5d555e82956c7deac15cd7d39a37 Mon Sep 17 00:00:00 2001
From: Mikita Lipski <mikita.lipski@amd.com>
Date: Wed, 23 Aug 2017 15:29:18 -0400
Subject: [PATCH 3491/5855] drm/amd/display: Add a clock source to a sharing
 pipe

Fix to allow DVI displays to share the same clock source

Change-Id: I181201e004aef029429d2bf0d90b45aab19c11fc
Signed-off-by: Mikita Lipski <mikita.lipski@amd.com>
Reviewed-by: Roman Li <Roman.Li@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc_resource.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
index cdf073d..b4ac29d 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
@@ -2536,7 +2536,7 @@ enum dc_status resource_map_clock_resources(
 		pipe_ctx->clock_source = NULL;
 
 		if (!dc->config.disable_disp_pll_sharing)
-			resource_find_used_clk_src_for_sharing(
+			pipe_ctx->clock_source = resource_find_used_clk_src_for_sharing(
 				&context->res_ctx,
 				pipe_ctx);
 
-- 
2.7.4

